core/transport/src/grpc/block_master.proto

```diff
@@ -98,6 +98,17 @@ message GetWorkerInfoListPResponse {
   repeated WorkerInfo workerInfos = 1;
 }
 
+message WorkerLostStorageInfo {
+  optional grpc.WorkerNetAddress address = 1;
+  /** a map from tier alias to the lost storage paths */
+  map<string, StorageList> lostStorage = 2;
```

Comment:
`optional`

Reply:
Grpc is different from Netty that `Field labels (required/optional/repeated) are not allowed on map fields.`, thanks